packages/core/src/index.ts

```diff
@@ -301,7 +301,7 @@ export const logHandledError = (
   return createFalsePromise();
 };
 
-export const startView = (view: string): Promise<boolean> => {
+export const startView = (view: string): Promise<string | boolean> => {
```

Comment:
would this be a breaking change for anyone consuming this and having used the result as a simple boolean or would typescript resolve the union in that case? If it is breaking we can maybe wait till 6.0 to change this as we'll probably want to remove these methods altogether in favour of working with spans directly